icons-gallery/src/components/Columns.tsx

```diff
@@ -0,0 +1,26 @@
+import { Devvit } from '@devvit/public-api';
+
+interface ColumnsProps {
```

Comment:
Suggest we move everything to types to avoid JSONValue issues when folks copy-and-paste. Eg:

```
type ColumnProps = {
  count: number
  children: JSX.Children
}
```